code/modules/mob/living/carbon/human/whisper.dm

```diff
@@ -66,13 +66,17 @@
 	var/spans = list(SPAN_ITALICS)
 	rendered = "<span class='game say'><span class='name'>[GetVoice()]</span>[alt_name] [whispers], <span class='message'>\"[attach_spans(message, spans)]\"</span></span>"
 
-	for(var/mob/M in listening)
-		M.Hear(rendered, src, languages, message, , spans)
+	for(var/atom/movable/AM in listening)
+		if(istype(AM,/obj/item/device/radio))
+			continue
+		AM.Hear(rendered, src, languages, message, , spans)

```

Comment:
Nope. Hear does not iterate over contents or anything like that, get_hearers does.